src/sentry/tasks/app_store_connect.py

```diff
@@ -71,17 +77,17 @@ def inner_dsym_download(project_id: int, config_id: str) -> None:
 
 
 def create_difs_from_dsyms_zip(dsyms_zip: str, project: Project) -> None:
-    with open(dsyms_zip, "rb") as fp:
-        created = debugfile.create_files_from_dif_zip(fp, project, accept_unknown=True)
-        for proj_debug_file in created:
-            logger.debug("Created %r for project %s", proj_debug_file, project.id)
+    with sentry_sdk.start_span(op="dsym-difs", description="Extract difs dSYM zip"):
```

Comment:
In local testing I've not been able to see this span show up yet, I'm not sure what is happening.